src/main/java/com/kodilla/ecommercee/domain/GroupEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "ProductGroup")
```

Comment:
celowo nazwałeś te entity tak odmiennie od nazwy klasy ?

Reply:
Tak - otóż kiedy zostawiłem "Group", lub w ogóle nic nie dodawałem, to okazało się, że "group" to jest deskryptor i wywalało błąd podczas kompilacji. Stąd uznałem, że trzeba to jakoś nazwać, ale inaczej niż "Group". Tylko dlatego tak to nazwałem.

Reply:
Ok